missing dependency on trajectory_msgs

The recent addition of trajectory_msgs as a dependency is not declared in the package.xml causing builds to fail for 0.33.1 in both kinetic and melodic.


e.g.: http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__mavros__ubuntu_xenial_amd64__binary/12/console
```
15:26:01 /usr/lib/ccache/x86_64-linux-gnu-g++   -DHAVE_SENSOR_MSGS_BATTERYSTATE_MSG -DROSCONSOLE_BACKEND_LOG4CXX -DROS_BUILD_SHARED_LIBS=1 -DROS_PACKAGE_NAME=\"mavros\" -Dmavros_plugins_EXPORTS -I/tmp/binarydeb/ros-kinetic-mavros-0.33.1/include -I/opt/ros/kinetic/include -I/opt/ros/kinetic/share/xmlrpcpp/cmake/../../../include/xmlrpcpp -I/opt/ros/kinetic/share/orocos_kdl/../../include -I/usr/include/eigen3  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2  -std=c++14 -fPIC   -o CMakeFiles/mavros_plugins.dir/src/plugins/setpoint_trajectory.cpp.o -c /tmp/binarydeb/ros-kinetic-mavros-0.33.1/src/plugins/setpoint_trajectory.cpp
15:26:06 /tmp/binarydeb/ros-kinetic-mavros-0.33.1/src/plugins/setpoint_trajectory.cpp:24:53: fatal error: trajectory_msgs/MultiDOFJointTrajectory.h: No such file or directory
15:26:06 compilation terminated.
```

--- a/mavros_extras/package.xml
+++ b/mavros_extras/package.xml
@@ -25,6 +25,7 @@
   <depend>mavros_msgs</depend>
   <depend>sensor_msgs</depend>
   <depend>std_msgs</depend>
+  <depend>trajectory_msgs</depend>
   <depend>visualization_msgs</depend>
   <depend>urdf</depend>
   <depend>tf</depend>

